docusaurus/docs/reactnative/basics/migrating-from-5.x-to-6.x.mdx

```diff
@@ -186,6 +186,13 @@ registerNativeHandlers({
 
 The type of `quotedMessage` is changed from `MessageType |  boolean` to `MessageType | undefined` for better in the `MessageInputContext`.
 
+#### Remove the deprecated code
+
+- We have removed `loadChannelAtMessage` from channel context because it was no more used.
+- We have removed the `handleDeleteMessage`, `handleEditMessage`, `handleQuotedReplyMessage`, `handleResendMessage`, `handleToggleBanUser` and `handleToggleMuteUser` from the message context because it was no more used.
+- Remove the `onMessageNew` prop from the `ChannelList` component. Use the `onNewMessage` prop instead.
+- Remove the `handleBlock` prop from the `Channel` component.
```

Comment:
can we add examples of alternatives here? ideally links to the docs.

Reply:
Done @oliverlaz